### DISCUSSION TITLE
Can't use copilot

### DISCUSSION THREAD

**Discussion by njtp111** [community/community#43681] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am a student within our GitHub Global Campus Program. But when I login from vscode，It reminds me that I need to pay.

![image](https://user-images.githubusercontent.com/43241287/211242989-593123d0-e3fc-499b-b567-b1beee423afb.png)

**Comment by otkrickey** [comment]
Solved: #43634
